src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -1664,468 +1661,1007 @@ static void vehicle_sprite_2_4(
-            vehicle_sprite_5_2(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedLeft45(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 3:
-            vehicle_sprite_5_3(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight22(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 4:
-            vehicle_sprite_5_4(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight45(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 5:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedLeft67(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 6:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedLeft90(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 7:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedLeft112(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 8:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedLeft135(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 9:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedLeft157(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 10:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight67(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 11:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight90(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 12:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight112(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 13:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight135(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 14:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25BankedRight157(session, vehicle, imageDirection, z, vehicleEntry);
             break;
         case 15:
-            vehicle_sprite_5_0(session, vehicle, imageDirection, z, vehicleEntry);
+            VehiclePitchUp25Unbanked(session, vehicle, imageDirection, z, vehicleEntry);
```

Comment:
I believe these bankings signify the vehicle should use a different car

Reply:
I believe these banks are only used for flying coaster versions of barrel rolls.

Reply:
> I believe these banks are only used for flying coaster versions of barrel rolls.

I checked and this seems to be correct, the regular barrel rolls have a special slope angle not a special bank angle.

Reply:
If these banks are implemented they should have the car switching like the banks on pitch 0, I think

Reply:
> If these banks are implemented they should have the car switching like the banks on pitch 0, I think

If that's true shouldn't it be a separate PR anyway? I'm very reluctant to make any changes to these lines without fully understanding what they do, because it might break stuff, and it's not related to the new draw code I'm adding here.